File: app/src/main/java/com/example/frontend_android/model/Bookings/BookingRequest.java
package com.example.frontend_android.model.Bookings;



import java.time.LocalDateTime;

public class BookingRequest {

    private Long projectId;

    private Long senderId;

    private Long recipientId;
    private String title;
    private String description;

    private LocalDateTime start;
    private LocalDateTime end;
}

File: app/src/main/java/com/example/frontend_android/model/Bookings/BookingRequest.kt
package com.example.frontend_android.model.Bookings

import java.time.LocalDateTime


data class BookingRequest(
    val projectId: Long,
    val dateMillis: Long?,
    val startHour: Int,
    val startMinute: Int,
    val endHour: Int,
    val endMinute: Int,
    val bookingTitle: String? = "Default",
    val bookingDescription: String,
    val userIds: List<Long> = emptyList()
)

